src/msw/settings.cpp

```diff
@@ -120,7 +120,7 @@ wxColour wxSystemSettingsNative::GetColour(wxSystemColour index)
         if ( index == wxSYS_COLOUR_MENUBAR )
         {
             BOOL isFlat ;
-            if ( SystemParametersInfo( SPI_GETFLATMENU , 0 ,&isFlat, 0 ) )
+            if ( wxSystemParametersInfo( SPI_GETFLATMENU , 0 ,&isFlat, 0 ) )
```

Comment:
This shouldn't depend on the monitor, so arguably this shouldn't be changed neither.